fortanix-vme/fortanix-vme-abi/src/lib.rs

```diff
@@ -67,10 +68,12 @@ pub enum Response {
     Connected {
         /// The vsock port the proxy is listening on for an incoming connection
         proxy_port: u32,
+        /// The address of the remote party
+        peer: Addr,
     },
     Bound {
-        /// The TCP port the parent VM is listening on
-        port: u16,
+        /// The local TCP address the parent VM is listening on
+        local: Addr,
```

Comment:
Warning: This is a breaking change. I think this is ok as we haven't released the code yet.